lib/ui/shared/styles.dart

```diff
@@ -125,6 +125,22 @@ class AppTextStyles {
     );
   }
 
+  //status icon
+  static Widget statusIcon(int count) {
```

Comment:
Can you remove the static keyword here and make the widget in a class, functional widgets are bad practice in flutter.